src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb

```diff
@@ -442,6 +442,1211 @@ Sum: 10, Count: 4")
 
         End Sub
 
+        <Fact()>
+        Public Sub Overloading001()
+            Dim comp = CreateCompilationWithMscorlibAndVBRuntime(
+<compilation>
+    <file name="a.vb"><![CDATA[
+
+Module m1
+    Sub Test(x as (a as integer, b as Integer))
+    End Sub
+
+    Sub Test(x as (c as integer, d as Integer))
+    End Sub
+End module
+
+]]></file>
+</compilation>, additionalRefs:={ValueTupleRef, SystemRuntimeFacadeRef})
```

Comment:
Consider extracting a static field for those common refs, as we did in the C# side.